src/Runner/Parallel/ReadonlyCacheManager.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Runner\Parallel;
+
+use PhpCsFixer\Cache\CacheManagerInterface;
+
+final class ReadonlyCacheManager implements CacheManagerInterface
+{
+    private CacheManagerInterface $cacheManager;
+
+    public function __construct(CacheManagerInterface $cacheManager)
+    {
+        $this->cacheManager = $cacheManager;
+    }
+
+    public function needFixing(string $file, string $fileContent): bool
+    {
+        return $this->cacheManager->needFixing($file, $fileContent);
+    }
+
+    public function setFile(string $file, string $fileContent): void {}
```

Comment:
not convinced we should have this in first place, and if we do - why to bother calling setFile at all (ie no exception `LogicException` here)

Reply:
Please take a look at [this commit's description](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/commit/60c72b1f7775af1e25c1e483cb57c3b0350bfc22) - does it address your concern?

Reply:
`ReadonlyCacheManager` was removed.